core/store/src/main/java/uk/gov/gchq/gaffer/store/library/GraphLibrary.java

```diff
@@ -216,4 +221,24 @@ private void checkExisting(final String graphId, final byte[] schema, final Stor
             }
         }
     }
+
+    private void checkSchemaExists(final Schema schema) {
+        if (null != getSchema(schema.getId())) {
+            if (!JsonUtil.equals(getSchema(schema.getId()).toJson(false), schema.toJson(false))) {
```

Comment:
Maybe you can use `JSONSerialiser.serialise()`

Reply:
.toJson() uses that method